File: Beginner/Python/Array/RemoveDuplicates.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        i = 0
        sum = len(nums)
        while i < sum - 1:
            if (nums[i] == nums[i+1]):
                nums.pop(i+1)
                sum -= 1
            else:
                i += 1
                
        return len(nums)

File: Beginner/Java/Array/RemoveDuplicates.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int size = 0, pos = 1;
        if (nums.length <= 1)
            return nums.length;
        while (pos < nums.length) {
            if (nums[size] == nums[pos])
                pos ++;
            else
                nums[++ size] = nums[pos ++];
        }
        return size + 1;
    }
}
